src/main/java/wooteco/subway/exception/SubwayException.java

```diff
@@ -0,0 +1,22 @@
+package wooteco.subway.exception;
+
+import org.springframework.http.HttpStatus;
+
+public class SubwayException extends RuntimeException {
```

Comment:
```suggestion
public abstract class SubwayException extends RuntimeException {
```

SubwayException은 상속으로만 사용하고 있는데, 계층을 위해서만 사용한다면 abstract를 통해 의도를 전달할 수 있습니다 :)
추가적으로 저는 모든 클래스를 abstract 또는 final로만 작성하는데 그 이유에 대해 고민해보면 좋을 것 같아요 ㅎㅎ

Reply:
오! 이 주제에 대해 다른 크루들과 얘기했던 적이 있었는데요, 단순히 final이 불변 객체의 이점을 가져가는 것 외에도 협업 관점에서 장점이 있다고 결론지었어요. abstract는 상속 가능한 클래스에만, 그 외에는 의도하지 않은 상황이 발생할 수 있으니 다른 모든 클래스는 final을 붙이면 좋겠다는 의견이었어요. 그렇다면 final/아무것도 사용하지 않은 클래스로도 의도를 전달할 수 있지 않냐는 말도 있었는데, 타인이 봤을 때 조금이라도 더 명확한 편이 좋지 않겠냐는 게 제 생각이었습니당!

Reply:
그렇다면 제 코드는 exception 말고 모두 final 클래스가 되겠군요..🤔